src/feed-worker.js

```diff
@@ -1,26 +1,35 @@
 const feedQueue = require('./feed-queue');
 const feedParser = require('./feed-parser');
+const parentLogger = require('../utils/logger');
+
+const log = parentLogger.child({ module: 'feed-worker' });
 
 exports.start = function () {
-  // Start processing jobs from the feed queue...
-  feedQueue.process(async (job) => {
-    const { url } = job.data;
-    const posts = await feedParser(url);
-    const processedPosts = [];
-    if (posts.length > 0) {
-      posts.forEach((post) => {
-        // We can extract any other infromation from the post that we need here.
-        const processedPost = {
-          author: post.author,
-          date: post.date,
-          title: post.title,
-          description: post.description,
-          postURL: post.link,
-        };
-        processedPosts.push(processedPost);
-      });
-      // We can pass these objects into another queue, For now just printing to the console.
-      console.log(processedPosts);
-    }
-  });
+  // for the sake of error handling and gracefull shutdown
+  try {
```

Comment:
I don't think wrapping this much code in a try catch is the right move.  We'd be better to do this in something like https://nodejs.org/api/process.html#process_event_uncaughtexception